service/policy/obligations/obligations.proto

```diff
@@ -0,0 +1,230 @@
+syntax = "proto3";
+
+package policy.obligations;
+
+import "common/common.proto";
+import "policy/objects.proto";
+import "policy/selectors.proto";
+import "google/protobuf/struct.proto";
+
+///
+/// Obligation
+///
+
+// Definitions
+message GetObligationRequest {
+    string id = 1;
+}
+
+message GetObligationResponse {
+    policy.Obligation obligation = 1;
+}
+
+message CreateObligationRequest {
+    // Required
+    string namespace_id = 1;
+    string name = 2;
+    // Optional
+    // Common metadata
+    common.MetadataMutable metadata = 100;
+}
+
+message CreateObligationResponse {
+    policy.Obligation obligation = 1;
+}
+
+message UpdateObligationRequest {
+    // Required
+    string id = 1;
+
+    // Optional
+    common.MetadataMutable metadata = 100;
+    common.MetadataUpdateEnum metadata_update_behavior = 101;
+}
+
+message UpdateObligationResponse {
+    policy.Obligation obligation = 1;
+}
+
+message DeleteObligationRequest {
+    string id = 1;
+}
+
+message DeleteObligationResponse {
+    policy.Obligation obligation = 1;
+}
+
+message ListObligationsRequest {
+    // Optional
+    // Namespace ID or name
+    string namespace = 1;
+
+    // Optional
+    policy.PageRequest pagination = 10;
+}
+
+message ListObligationsResponse {
+    repeated policy.Obligation obligations = 1;
+
+    policy.PageResponse pagination = 10;
+}
+
+// Values
+message GetObligationValueRequest {
+    string id = 1;
+}
+
+message GetObligationValueResponse {
+    policy.ObligationValue value = 1;
+}
+
+message CreateObligationValueRequest {
+    // Required
+    string obligation_id = 1;
```

Comment:
Should this be oneof `obligation_id` or `obligation_fqn`?

Reply:
If we only provide one, `id` is preferred, but you could make both options with the `identifier` approach: https://github.com/opentdf/platform/blob/cc169d969f0e3380a2341033bc53a1a0eece781a/service/policy/namespaces/namespaces.proto#L60-L75

Reply:
used identifier